forge-game/src/main/java/forge/game/ability/AbilityUtils.java

```diff
@@ -2903,31 +2903,20 @@ public static final List<SpellAbility> getBasicSpellsFromPlayEffect(final Card t
     public static final List<SpellAbility> getSpellsFromPlayEffect(final Card tgtCard, final Player controller, CardStateName state, boolean withAltCost) {
         List<SpellAbility> sas = new ArrayList<>();
         List<SpellAbility> list = new ArrayList<>();
-        collectSpellsForPlayEffect(list, tgtCard.getCurrentState(), controller, withAltCost);
+        collectSpellsForPlayEffect(list, tgtCard.getState(tgtCard.getCurrentStateName()), controller, withAltCost);
```

Comment:
no i mean use "CardStateName state" from the get param a few lines above

Reply:
hmn that will be a logic change? 🤔 
currently when facedown it would first add from that state and then from original

Reply:
okay then it seems to be right